cmd/thanos/receive.go

```diff
@@ -311,12 +316,15 @@ func runReceive(
 
 			// Before quitting, ensure the WAL is flushed and the DBs are closed.
 			defer func() {
+				level.Info(logger).Log("msg", "shutting down Multi TSDB")
 				if err := dbs.Flush(); err != nil {
-					level.Warn(logger).Log("err", err, "msg", "failed to flush storage")
+					level.Error(logger).Log("err", err, "msg", "failed to flush storage")
 				}
 				if err := dbs.Close(); err != nil {
-					level.Warn(logger).Log("err", err, "msg", "failed to close multi db")
+					level.Error(logger).Log("err", err, "msg", "failed to close storage")
+					return
 				}
+				level.Info(logger).Log("msg", "Multi TSDB is closed")
```

Comment:
Above you switched from multi TSDB to storage and here is MultiTSDB naming? Is it on purpose?

Reply:
Also worth to mention status of flush? I know it looks logic not but worth to check if there is any log on successful flush (:

Reply:
I'll unify them, you're right. I want to reserve `Mutli TSDB` for the package.